src/frontend-pwa/src/views/Report/report.styles.ts

```diff
@@ -8,35 +8,53 @@ import typography from '../../typography';
 
 export const StyledReportOuterDiv = styled.div`
     width: 100%;
-    min-height: 100svh;
-    // position: absolute;
     left:0;
     top:0;
```

Comment:
These values require `position: absolute` on the element being positioned and a `position: relative` on a parent element.

Reply:
As @Bottle7 mentioned, this container should still be styled using the position absolute, as its your main view container.

but for Bottle7, with the `absolute` it will work in the parent container as intended, all our views have this for the outmost container.